doc/api/v8.md

```diff
@@ -180,6 +180,9 @@ Returns an object with the following properties:
 * `does_zap_garbage` {number}
 * `number_of_native_contexts` {number}
 * `number_of_detached_contexts` {number}
+* `total_global_handles_size` {number}
```

Comment:
nit: Some of the existing fields are documented below, so I was wondering if we should document the newly added ones too, wdyt?

Reply:
By
> Some of the existing fields are documented below

I meant that the document elaborates on these 3 fields https://github.com/nodejs/node/blob/c6547c5362f39dd07cd2e609f34c7f586a3b222d/doc/api/v8.md?plain=1#L180-L182 in https://github.com/nodejs/node/blob/c6547c5362f39dd07cd2e609f34c7f586a3b222d/doc/api/v8.md?plain=1#L184-L196, i.e., we don't just mentioned the names, we also explain what those are.

I was wondering if we should also explain the newly added fields in this PR in a similar way.

Reply:
Thanks, i think we should explain it. please review again.